src/main/java/com/google/sps/servlets/GenerateServlet.java

```diff
@@ -58,10 +67,15 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     Set<Destination> filteredDestinations =
         filter(allDestinations, userPlaces, userDifficultyLevels);
 
-    writeToDataStore(filteredDestinations);
+    // Convert Destinations to Hunt Items, create Scavenger Hunt, store in Datastore
+    // TODO: Only return the amount of hunt items that the user wants
+    ArrayList<HuntItem> huntItems = convertToHuntItems(filteredDestinations);
+    ScavengerHunt scavHunt = new ScavengerHunt(huntItems, -1);
```

Comment:
You can make another constructor for ScavengerHunt  that only takes huntItems.  In that case, you can assume -1 for the index.

Side note: at some point we should probably discuss the idea of using index = 0 instead of -1.  I know a lot of the logic in go.js would need to change, but using -1 is not intuitive, since this should ideally just be a pointer to the "active" hunt item.